build.gradle.kts

```diff
@@ -52,4 +56,5 @@ dependencies {
 
   testImplementation("au.com.dius.pact.provider:junit5spring:4.1.14")
   testImplementation("com.h2database:h2:1.4.200")
+  testImplementation("uk.org.lidalia:slf4j-test:1.0.1")
```

Comment:
allows log messages to be stored in memory for a test, meaning assertions on logs can be made